### ISSUE TITLE
google chrome on webos

### ISSUE THREAD

**Issue by WBenz** [webosbrew/apps-repo#61]
is there a homebrew google chrome app for webos lg tv ?

**Comment by blackPantherOS** [comment]
yep, very good idea the Chrome, Firefox instead stupid chromium based suxx...

**Comment by mariotaku** [comment]
Not likely. LG TV has limited system spec, running a modern browser would be too heavy.

**Comment by frkca** [comment]
Hello,

has anyone made any efforts into this?

**Comment by JustablockCode** [comment]
we can find html-ised version of it

**Comment by team-orangeBlue** [comment]
one would not get far...

the chromium binaries for windows are already ~300mb and while that is manageable, performance would be a total loss
one would need about 1gb to run chromium.. decently I guess? and to be honest this isn't the nintendo 3ds where the browser makes you think "WHY DOES THIS EXIST IT DOESNT WORK"

you could get away with the stock browser if you really need one, it can run bilibili (if the app doesnt cut your needs, or filters videos, or smth else). sloppily, but it can.
![image](https://github.com/user-attachments/assets/21d55a6f-5d20-4b58-ae4b-53f88af1eeb8)

**Comment by frkca** [comment]
Thanks guys for the quick response. The builtin is what we're targeting now, sad thing is that on recent WebOS version for our HW (5.5.x) toggling "Always Show Address Bar" doesn't make any difference and we just need to get rid of the menu bar. Anyone of you had luck hiding/disabling it?

Thanks again!

**Comment by JustablockCode** [comment]
there is kinda chrome... that is what whole os runs on... at least "core" of it:

![Image](https://github.com/user-attachments/assets/721e1e50-ba08-4c9e-b0ff-49e1cbac73b7)

**Comment by frkca** [comment]
Would knowing the version help in some way? Something like the "--app" mode available at least for some of the versions?

**Comment by JustablockCode** [comment]
> Would knowing the version help in some way? Something like the "--app" mode available at least for some of the versions?

i think i know a way to put edge, chrome and other browsers... we can use api of https://browser.lol
